task/buildpacks-phases/0.1/buildpacks-phases.yaml

```diff
@@ -42,6 +42,9 @@ spec:
     - name: SOURCE_SUBPATH
       description: A subpath within the `source` input where the source to build is located.
       default: ""
+    - name: HOME_OVERRIDE
```

Comment:
Is there a reason why a user would want to change the Home override directory?

Reply:
I agree with this. Could this not be a parameter and instead just be an `empty-dir` volume mount?

Reply:
Good call.